tests/pages/firefox/desktop/customize.py

```diff
@@ -1,87 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I can still see references to this file in `tests/functional/newsletter/test_newsletter_embed.py`

Reply:
Is it weird that tests passed even with this outdated reference? Anyway, nice spot - will remove.

Reply:
Where did tests pass?

Reply:
Apparently I forgot functional tests do not run on PRs.